requirements/base.txt

```diff
@@ -30,3 +30,4 @@ python-dateutil==2.7.5
 ifcfg==0.17
 sqlalchemy==1.2.10
 user-agents==1.1.0
+sentry-sdk==0.6.9
```

Comment:
I am a little ambivalent about bundling this into our released builds.

Reply:
would it have to become a plugin?

Reply:
In general I'm a big fan of decoupling this and making it optional, but would need help implementing it as such

Reply:
Could have it not be in the requirements file at all, and make the import conditional (try/except)? Then, install it specifically in the demo environment? However, that would need to be coupled with either a) using something other than the PEX for spinning up a demo server (as we want to do anyway), or b) enabling external Python paths for PEX (as we also want to do anyway)